sphinx/builders/linkcheck.py

```diff
@@ -529,6 +529,15 @@ def run(self, **kwargs: Any) -> None:
                 if uri not in hyperlinks:
                     hyperlinks[uri] = uri_info
 
+        # raw nodes
+        for rawnode in self.document.findall(nodes.raw):
+            uri = rawnode.get('source')
+            if uri:
```

Comment:
Maybe the same check as images would be useful (`:// in uri`). A missing local file would cause the build to fail and is probably not worth checking.

Reply:
Added this check in 50b2d541bf0b083f0adcd4faf94a8c9b8dad5d51